web/src/components/apps/AppDetailPage.tsx

```diff
@@ -337,15 +337,15 @@ function AppDetailPage(props: Props) {
       const firstVersion = downstream.pendingVersions.find(
         (version: Version) => version?.sequence === 0
       );
-      if (firstVersion?.status === "unknown" && props.isEmbeddedCluster) {
+      if (
```

Comment:
would this ever be unknown now?

Reply:
i think we could remove that. probably left over from an early implementation?